.docker/Dockerfile.test

```diff
@@ -16,9 +16,8 @@ RUN mv /app/tests /tests
 RUN rm -rf /app
 COPY --from=cli /app /app
 
-# Copy the test scripts.
-RUN cp /app/vendor/govcms/scaffold-tooling/scripts/govcms-deploy /usr/local/bin/
-RUN cp /app/vendor/govcms/scaffold-tooling/scripts/tests/* /usr/local/bin/
+# Copy deploy and test scripts.
+RUN cp /app/vendor/govcms/scaffold-tooling/scripts/govcms* /usr/local/bin/
```

Comment:
The `scripts/tests/*` binaries were moved in https://github.com/govCMS/scaffold-tooling/commit/f58d5e90403786eac5d8ae7164b1741ecdbb59f0#diff-d6c5855a62cf32a4dadbc2831f0f295f

Like-for-like means we are only interested in the `govcms-*` binaries, not the commands in the `deploy` folder at this stage